src/static/bitbucket/components/App.tsx

```diff
@@ -22,18 +31,22 @@ type AppState = {
   bannerMessage: BannerMessage | null;
 };
 
-export class App extends React.Component {
+const qs = new URLSearchParams(window.location.search);
+
+export class App extends React.Component<{}, AppState> {
+  appName = qs.get('appName') || 'Landkid';
```

Comment:
Maybe there's a better default (i.e. point to the app name displayed above the addon)?

Reply:
I think this is fine tbh.